### ISSUE TITLE
Credit Card

### ISSUE THREAD

**Issue by jmccrae** [globalwordnet/english-wordnet#121]
13397185-n (Interlingual Index: i107009)
(n) credit card, plastic, charge card, charge plate 
> a card (usually plastic) that assures a seller that the person using it has a satisfactory credit rating and that the issuer will see to it that the seller receives payment for the merchandise delivered; "do you take plastic?"

Has the following hyponyms

13397599-n (Interlingual Index: i107010)
(n) bank card 
> a credit card issued by a bank

13397685-n (Interlingual Index: i107011)
(n) phone card, calling card 
> a card that is used instead of cash to make telephone calls

13397816-n (Interlingual Index: i107012)
(n) cashcard, cash card 
> a credit card that entitles the holder to receive cash

None of these are really "credit cards" and a new hypernym for all four should be introduced corresponding to https://en.wikipedia.org/wiki/Payment_card

**Comment by jmccrae** [comment]
New synset:

'payment card', 'plastic': a card (usually plastic) that assures a seller that the person using it has sufficient means of payment and that the issuer will see to it that the seller receives payment for the merchandise delivered; "do you take plastic?" (hyponym of `ewn-13396496-n`)

'credit card': a payment card that allow the consumers to build a continuing balance of debt, subject to interest being charged (see https://en.wikipedia.org/wiki/Credit_card) (hypnomy of previous)

Definition Updates: 

'bank card': a **payment** card issued by a bank

'cashcard': a **payment** card that entitles the holder to receive cash

Relation changes:

`ewn-13397937-n` (debit card) move hypernym to new synset above

**Comment by arademaker** [comment]
I feeling hard to follow suggestions using the GitHub issue only. we do need an interface or at least a better text format file. Another question is how these issues will evolve to PR? What is the criteria? 

Sorry for insisting on this meta-topic related to the way-to-go for this repo.

**Comment by jmccrae** [comment]
I can sketch the PR and push it to review before closing this issue

**Comment by jmccrae** [comment]
Please see linked PR
